WFP-VAM/prism-app
# Update download button for analysis to be able to download geoJson and csv

Linked to https://github.com/WFP-VAM/prism-app/issues/209

To test the feature:
- Generate an analysis
- Once it is generated, check that in the legend you have a multi-options button
- If you click on CSV or GeoJSON you should download the corresponding format
- In the analysis pop up check that the download button has been renamed DOWNLOAD CSV

![image](https://user-images.githubusercontent.com/44159363/203641865-cd1ec11b-c72b-48f9-a642-c5afed21ea03.png)
